keps/sig-network/3726-standard-application-protocols/README.md

```diff
@@ -176,6 +178,8 @@ type ServicePort struct {
   //
   // * Kubernetes-defined prefixed names:
   //   * 'kubernetes.io/h2c' - HTTP/2 over cleartext as described in https://www.rfc-editor.org/rfc/rfc7540
+  //   * 'kubernetes.io/websocket' - WebSocket over cleartext as described in https://www.rfc-editor.org/rfc/rfc6455
```

Comment:
above entry `h2c` is an abbreviation.
rfc defines these as `ws` and `wss` , should these abbreviations be used instead ?

Reply:
+1

Reply:
I'm good with using the abbreviation. Does anyone object?

Reply:
+1 to `wss` and `ws`